docs/components/movement-sensor/viam-visual-odometry.md

```diff
@@ -0,0 +1,253 @@
+    "rk1": 0.046535971648456166
+}
+```
+
+{{% alert title="Copy calibration data" color="note" %}}
+When you copy the calibration results into your `camera` component configuration, be sure to provide these values to the correct attributes in the target `camera` configuration.
+Specifically, note that the `height_px` and `width_px` attributes are not contained within the `intrinsic_parameters` array in the `camera` configuration, but are located outside of it.
+{{% /alert %}}
+
+## Attributes
+
+The following attributes are available to configure the `viam-visual-odometry` module:
+
+| Name | Type | Inclusion | Default | Description |
+| ---- | ---- | --------- | --------| ------------ |
+| `camera_name` | string | **Required** | | Camera name to be used for inferring the motion. |
+| `time_between_frames_s` | float | Optional | `0.1` | Target time between two successive frames, in seconds. Depending on the inference time and the time to get an image, the sleeping time after each inference will be auto-tuned to reach this target. Additionally, if the time between two successive frame is 5x larger than `time_between_frames_s`, another frame will be requested. This value depends on the speed of your system.|
+|`orb_n_features`| int | Optional | `10000` | Maximum number of features to retain. |
+|`orb_edge_threshold`| int | Optional | `31` | Size of the border where the features are not detected. It should roughly match the `orb_patch_size` attribute.  |
+|`orb_patch_size`| int | Optional | `31` | Size of the patch used by the oriented BRIEF descriptor.|
+|`orb_n_levels`| int | Optional | `8` |Number of pyramid levels.|
+|`orb_first_level`| int | Optional | `0` |Level of pyramid to put source image into.|
+|`orb_fast_threshold`| int | Optional | `20` | Fast threshold. |
+|`orb_scale_factor`| float | Optional | `1.2` | Pyramid decimation ratio, greater than 1. |
+|`orb_WTA_K`| int | Optional | `2` | Number of points that produce each element of the oriented BRIEF descriptor. |
+|`matcher`| string | Optional | `"flann"` | Either `"flann"` for [FLANN based matcher](https://docs.opencv.org/3.4/d5/d6f/tutorial_feature_flann_matcher.html) or `"BF"` for brute force matcher. The FLANN matcher will look for the two best matches using the KNN method so Lowe's ratio test can be performed afterward. The [brute force matcher](https://docs.opencv.org/4.x/dc/dc3/tutorial_py_matcher.html) uses Hamming norm. |
+|`lowe_ratio_threshold`| float | Optional | `0.8` | Threshold value to check if the best match is significantly better than the second best match. This value will not be used if brute force matcher is chosen. |
+| `ransac_prob` | float | Optional | `0.99` | Probability to find a subset without outliers in it. Defines the number of iterations to filter the outliers. The number of iterations is roughly given by `$k = \frac{\log(1-p)}{\log(1-w^n)}$`, where `$n$` is the number of points and `$w$` is the ratio of inliers to total points.|

```

Comment:
Might be nothing but when I built the doc with `make serve-dev-draft` the LateX formula for the number of iterations didn't render well like in [here](https://github.com/viamrobotics/viam-visual-odometry#attributes-description).

Reply:
Yes, I believe this is a dev build limitation. I will make sure it renders for prod publish (it should with existing tooling and default settings, but will double-check!). Appreciate the callout!

Reply:
I think you need to enable katex - haven't tried though. and I don't think you need backticks then
```suggestion
| `ransac_prob` | float | Optional | `0.99` | Probability to find a subset without outliers in it. Defines the number of iterations to filter the outliers. The number of iterations is roughly given by $k = \frac{\log(1-p)}{\log(1-w^n)}$, where `$n$` is the number of points and `$w$` is the ratio of inliers to total points.|
```